src/common/syncjournaldb.cpp

```diff
@@ -970,6 +976,45 @@ bool SyncJournalDb::setFileRecord(const SyncJournalFileRecord &_record)
     }
 }
 
+qint64 SyncJournalDb::lastSync()
```

Comment:
@FlexW Is it expected for a getter to return `qint64` while the setter accepts `quint64`?

Reply:
Thanks for catching. qint64 is right. Must be changed in the other place.